src/OpenFeature.Contrib.Hooks.Otel/MetricsHook.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using System.Reflection;
+using System.Threading.Tasks;
+using OpenFeature.Model;
+
+namespace OpenFeature.Contrib.Hooks.Otel
+{
+    /// <summary>
+    /// Represents a hook for capturing metrics related to flag evaluations.
+    /// The meter name is "OpenFeature.Contrib.Hooks.Otel".
+    /// </summary>
+    public class MetricsHook : Hook
+    {
+        private static readonly AssemblyName AssemblyName = typeof(MetricsHook).Assembly.GetName();
+        private static readonly string InstrumentationName = AssemblyName.Name;
+        private static readonly string InstrumentationVersion = AssemblyName.Version.ToString();
+
+        private readonly UpDownCounter<double> _evaluationActiveUpDownCounter;
```

Comment:
since the values will always be natural numbers, I suggest using `long`
```suggestion
        private readonly UpDownCounter<long> _evaluationActiveUpDownCounter;
```

Reply:
Yes please 👍

Reply:
Solved with https://github.com/open-feature/dotnet-sdk-contrib/pull/114/commits/f8e52e6d3631b87b936b8dc56534ec41d189477f